### PR TITLE
Conviva Integration Documentation

### PR DESCRIPTION
**Pull request by bjsignalfx**

### CODE REVIEW

File: conviva/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+>
+>![Example dashboard showing metrics from this plugin](././img/example_plugin_dashboard.png)
+
+#### Important conditions to watch out for
+
+The Conviva API response may return with filters listed in fields **filters_not_exist**, **filters_warmup** and **filters_incomplete_data**. This means data for these filters may be wholly or partially unavailable.
+
+Configuration validation will fail when a non metriclens enabled filter is listed in a metriclens metric configuration.
+
+The select all filters in combination with select all dimensions options should be used with care because of the potential to generate a tremendous amount of datapoints.
```
Comment (barbara-sfx) [inline conviva/README.md]: Does "select all filter" or "select all dimensions" need any capitalization?  

What are the ramifications of generating "a tremendous amount of datapoints"? (from the user's perspective) I mean, what could happen?

File: conviva/docs/gauge.attempts.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+title: attempts
+brief: Real-time/live time series of the number of attemps
```
Comment (barbara-sfx) [inline conviva/docs/gauge.attempts.md]: typo in both places:  attempts

File: conviva/docs/gauge.audience_metriclens_map.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+title: audience_metriclens_map
+brief: Real-time/live table of audience metricLens
```
Comment (barbara-sfx) [inline conviva/docs/gauge.audience_metriclens_map.md]: This is a dupe of the above

File: conviva/docs/gauge.avg_bitrate.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+---
+title: avg_bitrate
+brief: Real-time/live time series of the average bitrate
```
Comment (barbara-sfx) [inline conviva/docs/gauge.avg_bitrate.md]: I don't understand what you mean by time series, simple series, label series.  If users will know what this means, then never mind. But usually it's stuff like "the average bitrate" or "the number of concurrent plays" etc.

File: conviva/docs/gauge.ended_plays.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+---
+title: ended_plays
+brief: Real-time/live simple series of ended
```
Comment (barbara-sfx) [inline conviva/docs/gauge.ended_plays.md]: seems like a word is missing at the end

File: conviva/docs/gauge.ended_plays_timeseries.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+---
+title: ended_plays_timeseries
+brief: Real-time/live time series of ended
```
Comment (barbara-sfx) [inline conviva/docs/gauge.ended_plays_timeseries.md]: Dupe of above

File: conviva/docs/gauge.quality_metriclens_map.md [hunk 7]
```diff
@@ -0,0 +1,8 @@
+---
+title: quality_metriclens_map
+brief: Real-time/live table of quality metricLens
```
Comment (barbara-sfx) [inline conviva/docs/gauge.quality_metriclens_map.md]: Dupe of above

File: conviva/docs/gauge.rebuffering_ratio.md [hunk 8]
```diff
@@ -0,0 +1,8 @@
+---
+title: rebuffering_ratio
+brief: Real-time/live time series of rebuffered ratio
```
Comment (barbara-sfx) [inline conviva/docs/gauge.rebuffering_ratio.md]: "rebuffered ratio" ?

File: conviva/README.md [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+
+The Conviva monitor emits Live Conviva metrics. The image below shows a partial replication of the Conviva Pulse Real-Time Dashboard using Live Conviva metrics emitted by the monitor.
+
+![Real-Time Dashboard](./img/conviva_real-time_dashboard.png)
+
+#### Important conditions to watch out for
+
+The Conviva API response may return with filters listed in fields **filters_not_exist**, **filters_warmup** and **filters_incomplete_data**. This means data for these filters may be wholly or partially unavailable. When this happens it is logged in the SignalFx Smart Agent log file.
+
+Configuration validation will fail when a non MetricLens enabled filter is listed in a MetricLens metric configuration.
```
[the remark this answers is not in the document]
Comment (barbara-sfx) [reply conviva/README.md]: Also, use the same capitalization throughout, either metriclens or MetricLens

File: conviva/docs/gauge.avg_bitrate.md (hunk 4 above)
Comment (bjsignalfx) [reply conviva/docs/gauge.avg_bitrate.md]: Yes the user should know what time series, simple series and label series are.

File: conviva/README.md (hunk 1 above)
Comment (bjsignalfx) [reply conviva/README.md]: Ah, I took out the word "tremendous" in the README.md version I am asking to be reviewed. I'm sure what's happening.

**Pull request merged**